common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.h

```diff
@@ -113,7 +113,8 @@ class mon_wlan_hal_dwpal : public base_wlan_hal_dwpal, public mon_wlan_hal {
     }
 
     std::shared_ptr<char> m_temp_dwpal_value;
-    uint32_t m_nl_seq = 0;
+    uint32_t m_nl_seq            = 0;
```

Comment:
Can you please add a comment explaining this member? Its name is not so self-explanatory.